src/sqlfluff/dialects/dialect_mariadb_keywords.py

```diff
@@ -0,0 +1,706 @@
+"""A List of MariaDB SQL keywords.
+
+https://mariadb.com/kb/en/reserved-words/
+"""
```

Comment:
I added slightly more documentation to my list of reserved keywords, especially where to get the unreserved keywords from. Feel free to steal it :)

https://github.com/pprkut/sqlfluff/blob/mariadb-dialect/src/sqlfluff/dialects/dialect_mariadb_keywords.py#L1

Reply:
Thanks! Adding in upcoming commit :)